### PR TITLE
feat: Display service status in the app

### PR DESCRIPTION
**Pull request by klochowicz**
- Add the status to the AppBar, next to Settings screen
- display service status as well as overall status

TODO:
  - [ ] Colour-coding for the status?
  - [ ] Status icon should be a circle coloured with the value of enum

Posting as-is as I suspect that this will spark the conversation whether we'd
like to have a hamburger menu or not instead of row of icons.

https://github.com/get10101/10101/assets/8319440/459362cb-a2ba-4572-8f17-23f13ad5b3a6

only difference since the video: "overall" -> "App Health"

### CODE REVIEW

**Comment by bonomat** [comment]
For frontend changes always add a screenshot please.

File: mobile/lib/common/status_screen.dart [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      appBar: AppBar(title: const Text("Status")),
+      body: SafeArea(
+        child: Column(
+          children: [
+            const SizedBox(height: 20),
+            Text("Overall: $overallStatus"),
+            const Divider(),
+            Text("Orderbook: $orderbookStatus"),
+            const SizedBox(height: 10),
+            Text("Coordinator: $coordinatorStatus")
```
Comment (da-kami) [inline mobile/lib/common/status_screen.dart:36]:
```suggestion
            Text("LSP: $coordinatorStatus")
```

Might be better to call the "coordinator" LSP or Trade Service for the user?

File: mobile/lib/common/status_screen.dart [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
+    final overallStatus = statusToString(serviceStatusNotifier.overall());
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Status")),
+      body: SafeArea(
+        child: Column(
+          children: [
+            const SizedBox(height: 20),
+            Text("Overall: $overallStatus"),
```
Comment (da-kami) [inline mobile/lib/common/status_screen.dart:32]: 🙃 Might be worth using our `ValueDataRow` row with `ValueType.text` here.

File: mobile/lib/common/status_screen.dart [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+    final orderbookStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Orderbook));
+    final coordinatorStatus =
+        statusToString(serviceStatusNotifier.getServiceStatus(Service.Coordinator));
+    final overallStatus = statusToString(serviceStatusNotifier.overall());
+
+    return Scaffold(
+      appBar: AppBar(title: const Text("Status")),
+      body: SafeArea(
+        child: Column(
```
Comment (da-kami) [inline mobile/lib/common/status_screen.dart:29]: 🙃 might be worth adding some form of centered icon to the screen that shows the overall status in green, red or gray.

**Review by da-kami — APPROVED** [review]
I think it's good to merge, screenshot would be nice. Improvements can be added on top.

File: mobile/lib/common/status_screen.dart (hunk 3 above)
Comment (klochowicz) [reply mobile/lib/common/status_screen.dart]: I plan to do that in a follow-up!

**Pull request merged**